lib/stdlib/src/base64.erl

```diff
@@ -512,13 +542,60 @@ b64d(X) ->
              bad,bad,bad,bad,bad,bad,bad,bad,bad,bad,bad,bad,bad,bad,bad,bad,
              bad,bad,bad,bad,bad,bad,bad,bad,bad,bad,bad,bad,bad,bad,bad,bad,
              bad,bad,bad,bad,bad,bad,bad,bad,bad,bad,bad,bad,bad,bad,bad,bad,
+             bad,bad,bad,bad,bad,bad,bad,bad,bad,bad,bad,bad,bad,bad,bad,bad,
+
+	     % alternative base64url alphabet (RFC 4648 Section 5)
```

Comment:
```suggestion
	     %% alternative base64url alphabet (RFC 4648 Section 5)
```

and similarly for other `%` comments in similar positions (i.e. not comments at the end of a line).